cohortextractor/__main__.py

```diff
@@ -36,6 +36,13 @@ def existing_python_file(value):
     help="Provide dummy data from a file to be validated and used as output",
     type=Path,
 )
+generate_cohort_parser.add_argument(
+    "--validate-backend",
+    type=str,
+    nargs="?",
```

Comment:
I'm not v familiar with `argparse`, but this `?` looks odd. Don't we always want exactly one value?

Reply:
`?` means 0 or 1, to make it optional.  This will go away if it becomes a separate command though

Reply:
`--validate-backend` is optional, but doesn't it always require a backend to be specified when it's present? I understood that `?` would make the backend optional, not the argument itself.

Reply:
In this case it would have the same effect, since we check if there is a backend specified.  This is gone now, though